ARGOS_service_data_converter.py

```diff
@@ -151,7 +154,7 @@ def __init__(self, missing=1e35):
       self.missing = missing
 
     @staticmethod
-    def get_data(fobj=None, time='current'):
```

Comment:
This change will break the API and it is needed to get current data without specifying a date. Put it back in and resubmit.

Reply:
I didn't do this so something is weird.

Reply:
its not really worth investigating on such a small program probably it was incidental or you copy pasted from a old file or something... doesn't matter, its small enough, i'll accept the changes and then fix it myself and push another commit.  It was useful enough to get a feel how to handle pull requests.